products/editions/enterprise-edition/b2b-suite/guides/storefront/ajax-panel.md

```diff
@@ -0,0 +1,134 @@
+# Ajax panel
+
+You can download a plugin showcasing the topic [here](/products/editions/enterprise-edition/b2b-suite/example-plugins/B2bAjaxPanel.zip).
+
+## Table of contents
+
+* [Description](#description)
+* [Basic usage](#basic-usage)
+* [Extended usage](#extended-usage)
+  * [Make links clickable](#make-links-clickable)
+  * [Ignore links](#ignore-links)
+  * [Link to a different panel](#link-to-a-different-panel)
+* [Ajax Panel Plugins](#ajax-panel-plugins)
+  * [Modal](#modal)
+  * [TriggerReload](#triggerreload)
+  * [TreeSelect](#treeselect)
+
+## Description
+
+`AjaxPanel` is a jQuery based extension to the Shopware Frontend Framework. It mimics `iFrame` behaviour by integrating content from different controller actions through ajax into a single view and intercepting, usually page changing, events and transforming them into XHR-Requests.
```

Comment:
"jQuery based" is not up to date. You could replace this with "... is a mini-framework based on storefront plugins."